js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let midImg = document.getElementById("cta-img");
+midImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let lastImg = document.getElementById("middle-img");
+lastImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+const navBar = document.querySelector("nav").children;
+navBar[0].textContent = siteContent["nav"]["nav-item-1"];
+navBar[1].textContent = siteContent["nav"]["nav-item-2"];
+navBar[2].textContent = siteContent["nav"]["nav-item-3"];
+navBar[3].textContent = siteContent["nav"]["nav-item-4"];
+navBar[4].textContent = siteContent["nav"]["nav-item-5"];
+navBar[5].textContent = siteContent["nav"]["nav-item-6"];
+navBar[0].style.color = "green";
+navBar[1].style.color = "green";
+navBar[2].style.color = "green";
+navBar[3].style.color = "green";
+navBar[4].style.color = "green";
+navBar[5].style.color = "green";
+
+const newNav = document.createElement("a");
+const nav = document.querySelector("nav");
+nav.appendChild(newNav);
+newNav.textContent = "Instagram"
+newNav.style.color = "green";
+
+const newNav2 = document.createElement("a");
+const hi = document.querySelector("nav");
+hi.prepend(newNav2);
+newNav2.textContent = "Facebook"
+newNav2.style.color = "green";
+
+
+const topHeader = document.querySelector("h1");
+topHeader.textContent = siteContent["cta"]["h1"];
+topHeader.innerHTML = "DOM<br> IS<br> AWESOME";
+
+const button = document.querySelector("button");
+button.textContent = siteContent["cta"]["button"];
+
+const midContent = document.querySelectorAll("p");
+midContent[0].textContent = siteContent["main-content"]["features-content"];
+midContent[1].textContent = siteContent["main-content"]["about-content"];
+midContent[2].textContent = siteContent["main-content"]["services-content"];
+midContent[3].textContent = siteContent["main-content"]["product-content"];
+midContent[4].textContent = siteContent["main-content"]["vision-content"];
+
+
+const midHeaders = document.querySelectorAll("h4");
+midHeaders[0].textContent = siteContent["main-content"]["features-h4"];
+midHeaders[1].textContent = siteContent["main-content"]["about-h4"];
+midHeaders[2].textContent = siteContent["main-content"]["services-h4"];
+midHeaders[3].textContent = siteContent["main-content"]["product-h4"];
+midHeaders[4].textContent = siteContent["main-content"]["vision-h4"];
+
+const contactHeader = document.querySelectorAll("h4");
+contactHeader[5].textContent = siteContent["contact"]["contact-h4"];
+
+const contactStuff = document.querySelectorAll("p");
+contactStuff[5].textContent = siteContent["contact"]["address"];
+contactStuff[6].textContent = siteContent["contact"]["phone"];
+contactStuff[7].textContent = siteContent["contact"]["email"];
+
```

Comment:
all of these three variables are fair game to use a for loop or array method.  remember the DRY principle.  "Do Not Repeat"
But totally works, and great job getting everything to work.  sometimes reps are the way to go while you're learning.